README.npm.md

```diff
@@ -0,0 +1,195 @@
+[![License](https://img.shields.io/badge/license-Apache%202.0-3da639.svg)](https://opensource.org/licenses/Apache-2.0)
+[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg)](https://github.com/prettier/prettier)
+[![Slack Organisation](https://img.shields.io/badge/slack-chat-blueviolet.svg?label=Kedro%20Slack&logo=slack)](https://slack.kedro.org)
+
+## Introduction
+
+Kedro-Viz is an interactive development tool for building data science pipelines with [Kedro](https://github.com/kedro-org/kedro). Kedro-Viz also allows users to view and compare different runs in the Kedro project.
+
+## Features
+
+- ✨ Complete visualisation of a Kedro project and its pipelines
+- 🎨 Supports light & dark themes out of the box
+- 🚀 Scales to big pipelines with hundreds of nodes
+- 🔎 Highly interactive, filterable and searchable
+- 🔬 Focus mode for modular pipeline visualisation
+- 📊 Rich metadata side panel to display parameters, plots, etc.
+- 📊 Supports all types of [Plotly charts](https://plotly.com/javascript/)
+- ♻️ Autoreload on code change
+- 🧪 Supports tracking and comparing runs in a Kedro project
+- 🎩 Many more to come
+
+## Installation
+
+- As a standalone React component (for embedding Kedro-Viz in your web application).
+
+  To install the standalone React component:
+
+  ```bash
+  npm install @quantumblack/kedro-viz
+  ```
+
+## Usage
+
+To use Kedro-Viz as a standalone React component, you can follow the example below. Please note that Kedro-Viz does not support server-side rendering (SSR). If you are using Next.js or another SSR framework, you should be aware of this limitation.
+
+```javascript
+import KedroViz from '@quantumblack/kedro-viz';
+import '@quantumblack/kedro-viz/lib/styles/styles.min.css';
+
+const MyApp = () => <KedroViz data={json} />;
+```
+
+To use with NextJS:
+
+```javascript
+import '@quantumblack/kedro-viz/lib/styles/styles.min.css';
+import dynamic from 'next/dynamic';
+
+const NoSSRKedro = dynamic(() => import('@quantumblack/kedro-viz'), {
+  ssr: false,
+});
+
+const MyApp = () => <NoSSRKedro data={json} />;
+```
+
+The JSON can be obtained by running:
+
+```bash
+kedro viz run --save-file=<path-to-save-kedro-viz-data-to-a-directory>
+```
+
+On successful execution of the command above, it will generate a folder at the specified directory, containing the following structure: 
+
+```
+<filename>/api/
+├── main
+├── nodes
+│   ├── 23c94afb
+│   ├── 28754fab
+│   ├── 2ab3579f
+│   ├── 329e963c
+│   ├── 369acf98
+│   └── ...
+└── pipelines
+    ├── __default__
+    ├── data_processing
+    ├── data_science
+    └── ...
+```
+
+Please use the `main` file as the input JSON for the `data` prop in your Kedro-Viz component.
+
+We also recommend wrapping the `Kedro-Viz` component with a parent HTML/JSX element that has a specified height (as seen in the above example) in order for Kedro-Viz to be styled properly.
```

Comment:
I don't see the example where it's wrapped in a parent element. Am I missing it?

Reply:
Thanks for catching, yes its not wrapped.
Is it good idea to show wrap in below example?
```
import KedroViz from '@quantumblack/kedro-viz';
import '@quantumblack/kedro-viz/lib/styles/styles.min.css';

const MyApp = () => <KedroViz data={json} />; 
```
  
  with
  
```
import KedroViz from '@quantumblack/kedro-viz';
import '@quantumblack/kedro-viz/lib/styles/styles.min.css';

const MyApp = () => <div style={{height: `100vh`}}><KedroViz data={json} /></div>; 
```

or create a separate code snippets with below text.
"We also recommend wrapping the `Kedro-Viz` component with a parent HTML/JSX element that has a specified height (as seen in the above example) in order for Kedro-Viz to be styled properly."

Reply:
I think the second code example is what should be shown in the docs. That's the recommended way of doing it, as far as I still understand it, right? If so, then show that.